content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,119 @@
+
+Let's not forget the PnP (Plug n' Play) manager of the kernel which is also an important part not only in regard of stability the usability of the system to a plethora of hardware.
+Vadim Galyant added the initial headers implementation of the Resources Arbiter library which can be used by the bus
+drivers like ACPI, PCI or legacy PIC HAL. In addition to that Vadim implemented debug code both in PnP and I/O managers whereas Eric Kohl improved the PnP manager to map device capabilities to capability flags. Thomas Faber fixed a critical bug that could
+cause memory corruption in the kernel space and fixed a bad IRP (I/O request packet) handling that caused a BSoD (Blue Screen of Death) on first generation Xbox console machine with USB enabled. 
+
+The Kernel Debugger (KD) and related modules have also received updates from developers and contributors alike. Dmitry Borisov added ComPort library for NEC PC-98 which is fundemantal in the further development and support of NEC series. Hermès Bélusca-Maïto has finished the support for debug filters, improved the `cregs` and `tss` commands included in the interactive kernel debugger (KDBG, present only in GCC debug builds),
+rewrote the TSS handling code and squashed various bugs with it. Namely the issue with the KDBG backtrace when a TSS switch happened during execution. Furthermore, Jérôme Gardou alongside with Hervé Poussineau brought some fixes and improvements to the KDBG module.
+
+## NetKVM VirtIO bringup
+NetKVM VirtIO is an NDIS 5 compatible driver developed by Red Hat Inc., which allows fast VirtIO networking in virtual machine software such as QEMU and other KVM hardware accelerated virtual machines.
+Nguyen Trung Khanh, a ReactOS contributor, took over Benjamin Aerni's work of importing the driver into the source tree of ReactOS.
+This provides ReactOS with support for another network interface out of the box.
+
+## Miscellaneous changes & improvements
+Other areas of ReactOS infrastructure have received lots improvements, ranging from device drivers to software applications of ReactOS by various developers and contributors. Notably FreeLoader, the ReactOS' bootloader, received incremental fixes and further support improvements. As such Hermès Bélusca-Maïto added support for booting Linux 64-bit systems in 64-bit FreeLoader and fixed an issue where
+FreeLoader couldn't read from an EXT volume hence preventing booting. Dmitry Borisov fixed a serious triple fault bug when ReactOS was booted up in Screen mode. In addition to that, Dmitry Borisov also provided other fixes as well as adding ARC-emulation support necessary for NEC PC-98 series.
+Mark Harmstone also brought some fixes for the bootloader so that Windows Vista can be properly booted. On-Screen Keyboard and Accessbility Utility Manager are being regularly updated by Bisoc George and at the same time he provided some fixes or minor improvements to other parts of ReactOS.
```

Comment:
```suggestion
George Bișoc updated the On-Screen Keyboard and Accessbility Utility Manager and at the same time provided minor fixes and improvements to different parts of ReactOS.
```

Reply:
Why stripping that Mark Harmstone Vista boot stuff?

Reply:
It was not merged - PRs are still there

Reply:
Nice catch, in fact it's not merged yet even now.

Reply:
because part of it is controversial (like the correct place where to reserve a hole in the memory, as well as supporting/handling the different version-specific layouts of the `LOADER_PARAMETER_BLOCK` structure).